packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -81,6 +81,10 @@ const propTypes = {
    * An ISO 8601 string representation of the initial value to show in the date and time inputs.
    */
   value: PropTypes.string,
+  /**
+   * Type of time input to initialize. Must be '24-hour' or '12-hour'
+   */
+  timeVariant: PropTypes.oneOf([DateTimeUtils.FORMAT_12_HOUR, DateTimeUtils.FORMAT_24_HOUR]),
```

Comment:
I think this new prop deserves to have an example [here](https://engineering.cerner.com/terra-ui/#/components/terra-date-time-picker/date-time-picker/date-time-picker) to show how the 12-hour format looks like.

Reply:
added example for [timeVariant](https://github.com/cerner/terra-framework/blob/44faa6c1df1f928ccce787ef4cded5943b3c46e5/packages/terra-date-time-picker/src/terra-dev-site/doc/date-time-picker/DateTimePicker.1.doc.jsx#L138) in date-time-picker docs